src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.mm

```diff
@@ -120,7 +124,40 @@ void SetDelegate(id<MTROTAProviderDelegate> delegate, dispatch_queue_t delegateN
         }
     }
 
+    void ResetState()
```

Comment:
Why is this public?  It doesn't need to be public...  Please put this back where it used to be, in the private methods.